_build_cfg.yml

```diff
@@ -243,6 +243,8 @@ Topics:
     File: building_dependency_trees
   - Name: Troubleshooting Networking
     File: sdn_troubleshooting
+  - Name: Downgrading to 3.0
+    File: downgrade
```

Comment:
Since this is specific to OSE, we'll need to add a distro after the `File` here so that the topic only shows up in the OSE docs and not Origin.

```
  - Name: Downgrading to 3.0
    File: downgrade
    Distros: openshift-enterprise
```